src/main/java/org/jabref/logic/importer/fetcher/ZbMATH.java

```diff
@@ -86,4 +88,9 @@ public void doPostCleanup(BibEntry entry) {
         new FieldFormatterCleanup(StandardField.JOURNAL, new RemoveBracesFormatter()).cleanup(entry);
         new FieldFormatterCleanup(StandardField.TITLE, new RemoveBracesFormatter()).cleanup(entry);
     }
+
+    @Override
+    public AbstractQueryTransformer getQueryTransformer() {
+        return new ZbMathQueryTransformer();
```

Comment:
I would put these transformer classes as private inner classes. I think it's an implementation detail how the fetcher transforms the user-query to a query used in the url.

Reply:
That shouldn't be a problem. You just move the current code into the fetcher class.